lib/graylog2_exceptions.rb

```diff
@@ -165,7 +165,25 @@ def clean_stack(backtrace)
         line.gsub! gem_root_str, "[GEM_HOME]/"
       end
     end
-    arr.join("\n")
+    arr
+  end
+
+  def clean_stack(backtrace)
+    backtrace_cleaner.clean(backtrace).join("\n")
+  end
+
+  def backtrace_cleaner
+    @backtrace_cleaner ||=get_backtrace_cleaner
+    @backtrace_cleaner
+  end
+
+  def get_backtrace_cleaner
```

Comment:
I'd implement it in backtrace_cleaner. There's no real reason to do it in a different method